Bottom: a7cdbd886f6b45cc3e2971a4c9021c19b4368188
Top:    e73aca75bbb437e2fdf00c62d2cf1297494c585d
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-14 17:23:16 +0800

autofs-5.0.7 - fix teach dumpmaps to output simple key value pairs


---

diff --git a/daemon/automount.c b/daemon/automount.c
index 72b284c..97c726a 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1725,7 +1725,8 @@ static void usage(void)
 		"	-f --foreground do not fork into background\n"
 		"	-r --random-multimount-selection\n"
 		"			use ramdom replicated server selection\n"
-		"	-m --dumpmaps	dump automounter maps and exit\n"
+		"	-m --dumpmaps [<map type> <map name]"
+		"			dump automounter maps and exit\n"
 		"	-n --negative-timeout n\n"
 		"			set the timeout for failed key lookups.\n"
 		"	-O --global-options\n"
